front/ui/ui-core/src/components/inputs/ComboBox/ComboBox.tsx

```diff
@@ -25,6 +25,13 @@ export type ComboBoxProps<T> = Omit<InputProps, 'value'> & {
   onSelectSuggestion: (option: T | undefined) => void;
   resetSuggestions: () => void;
   testIdPrefix?: string;
+  listElementComponent?: (params: {
```

Comment:
`listElementComponent` is actually a function which returns a component, not a component itself
```suggestion
  renderListElementComponent?: (params: {
```